code/controllers/subsystem/priority_directives.dm

```diff
@@ -0,0 +1,102 @@
+SUBSYSTEM_DEF(directives)
+	name = "Priority Directives"
+	wait = 10 SECONDS
```

Comment:
Objectives are created randomly every 20-30 minutes, and the wait of subsystems by definition isn't precise due to how we need to schedule all subsystems to use limited time.
The expiration of directives is tracked with a very verbose timer on the uplink, so I don't think this will be able to be used to break immersion in any way. TGUI updates are still performed every second, as they are tied to SStgui, the directives simply determine when a directive should spawn and when a directive should be considered complete.